src/db.js

```diff
@@ -43,6 +43,27 @@ class Db {
       .then(ref => ref.data())
       .catch(() => null);
   }
+
+  async joinParty(party) {
+    const { deviceId } = Constants;
+    const timestamp = Date.now();
```

Comment:
Rather than using `timestamp` as a name, we could use a more descriptive one like `joinedAt`.

Reply:
I agree. I'm referencing #36 to make it count for the brainstorming.